hazelcast/src/test/java/com/hazelcast/cache/CacheBasicAbstractTest.java

```diff
@@ -534,7 +500,7 @@ public void testJSRCreateDestroyCreate()
         Cache<Integer, String> cache = cacheManager.createCache(cacheName, config);
         assertNotNull(cache);
 
-        Thread.sleep(1000);
+        sleepSeconds(1);
```

Comment:
@Donnerbart safe to remove. It is left over. Cache config creation on other nodes is changed to sync by not blocking partition threads long time ago.